hypothesis-python/src/hypothesis/internal/intervalsets.py

```diff
@@ -35,7 +35,7 @@ def from_string(cls, s: str) -> "Self":
         >>> IntervalSet.from_string('abcdef0123456789')
         ((48, 57), (97, 102))
         """
-        x = cls((ord(c), ord(c)) for c in sorted(s))
+        x = cls([(ord(c), ord(c)) for c in sorted(s)])
```

Comment:
I'm surprised we needed this, but 🤷

Reply:
ah, it's not required...this is for mypyc, which [interprets generator expressions as list comprehensions](https://mypyc.readthedocs.io/en/latest/differences_from_python.html#generator-expressions). I hadn't meant to bring any mypyc-specific changes over yet, but I'll just leave this one in.